Remove-Labs.ps1

```diff
@@ -10,19 +10,32 @@ $ErrorActionPreference = "Stop"
 # Common setup for scripts
 . "./Utils.ps1"                                          # Import all our utilities
 Import-AzDtlModule                                       # Import the DTL Library
-$config = Import-ConfigFile -ConfigFile $ConfigFile      # Import all the lab settings from the config file
+$labConfig = Import-ConfigFile -ConfigFile $ConfigFile   # Import all the lab settings from the config file
+$labConfigCount = ($labConfig | Measure-Object).Count
 
-Write-Host "---------------------------------" -ForegroundColor Green
-Write-Host "Removing the following $(($config | Measure-Object).Count) labs from Azure..." -ForegroundColor Green
+$bastionLabConfig = [Array] ($labConfig | Where-Object { $_.BastionEnabled })
+$bastionLabConfigCount = ($bastionLabConfig | Measure-Object).Count
 
-$config | Select-Object DevTestLabName, ResourceGroupName | Format-Table | Out-String | Write-Host
+if ($bastionLabConfigCount -gt 0) {
+    Write-Host "---------------------------------" -ForegroundColor Green
+    Write-Host "Removing Bastion hosts from the following $bastionLabConfigCount labs..." -ForegroundColor Green
+    $labConfig | Select-Object DevTestLabName, ResourceGroupName | Format-Table | Out-String | Write-Host
```

Comment:
Line 22 - instead of "$labConfig" as what you write out, shouldn't it be $bastionLabConfigCount?